client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -32,7 +33,11 @@ function isSkippedTaxonomy( postType, taxonomy ) {
 	return false;
 }
 
-function EditorDrawerTaxonomies( { translate, siteId, postType, taxonomies, terms } ) {
+function EditorDrawerTaxonomies( { translate, siteId, postType, isSupported, taxonomies, terms } ) {
+	if ( ! isSupported ) {
+		return null;
```

Comment:
Yeah, probably so. But better to at least hide for now than to mislead the user to think they can manage terms. I'll go ahead and merge this one now and we can plan to enhance it separately.